### PR TITLE
add bobholt and jgraham as tools/wptrunner owners

### PR DESCRIPTION
**Pull request by bobholt**
This change is [Reviewable](https://reviewable.io/reviews/w3c/web-platform-tests/6034)

### CODE REVIEW

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/235597712)

# Firefox (nightly) #
Testing web-platform-tests at revision 313974f367746f50f5a926470bddab62bd2ca2e4
Using browser at version BuildID 20170523100217; SourceStamp 5bc1c758ab57c1885dceab4e7837e58af27b998c
Starting 10 test iterations
No tests run.

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/235597713)

# Sauce (safari) #
Testing web-platform-tests at revision 313974f367746f50f5a926470bddab62bd2ca2e4
Using browser at version 10.0
Starting 10 test iterations
No tests run.

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/235597722)

# Chrome (unstable) #
Testing web-platform-tests at revision 313974f367746f50f5a926470bddab62bd2ca2e4
Using browser at version 60.0.3107.4 dev
Starting 10 test iterations
No tests run.

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/235597723)

# Sauce (MicrosoftEdge) #
Testing web-platform-tests at revision 313974f367746f50f5a926470bddab62bd2ca2e4
Using browser at version 14.14393
Starting 10 test iterations

File: tools/wptrunner/OWNERS [hunk 1]
```diff
@@ -0,0 +1,2 @@
+@bobholt
+@jgraham
```
Comment (jgraham) [inline tools/wptrunner/OWNERS]: I own the whole of `tools/` so this isn't really necessary.

Comment (bobholt) [inline tools/wptrunner/OWNERS]: Ah. Couldn't remember if we settled on notifications being recursive or fallback. Updated.

**Review by gsnedders — APPROVED** [review]

**Pull request merged**